src/templates/globals/util/HTMLComment.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react'
+
+interface HTMLCommentProps {
+  position: 'head' | 'footer'
```

Comment:
i know these are the only two places we've had to use comments, but it feels kind of limiting. i'd really prefer something more generic... but we can cross that bridge when we come to it, this should be fine for our purposes for now.

Reply:
I agree that its limiting but I believe there will be some extra consideration around cleanup for adding many more comments, and to the body.  I don't think html comments are something we will need to reach for very often though.